src/components/Card.spec.js

```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react'
+import Card from './Card'
+
+describe('Card', () => {
+  it('renders name of card', () => {
+    render(<Card title="Völkerschlachtdenkmal" />)
+    expect(screen.getByText('Völkerschlachtdenkmal')).toBeInTheDocument()
+  })
+  it('renders text of card', () => {
+    render(<Card text="heading" />)
+    expect(screen.getByRole('heading')).toBeInTheDocument()
+  })
```

Comment:
This does _not_ test the text! When you query be `getByRole('heading')` you get the `h2` element of your card.